program/rust/src/accounts/price.rs

```diff
@@ -302,3 +303,82 @@ impl PriceFeedMessage {
         bytes
     }
 }
+
+/// Message format for sending Twap data via the accumulator program
+#[repr(C)]
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub struct TwapMessage {
+    pub id:                [u8; 32],
+    pub cumulative_price:  i128,
+    pub cumulative_conf:   u128,
+    pub num_down_slots:    u64,
+    pub exponent:          i32,
+    pub publish_time:      i64,
+    pub prev_publish_time: i64,
```

Comment:
this is to make sure we get the unique one per second?

Reply:
Yes